src/test/java/uk/gov/hmcts/cmc/claimstore/events/claim/DocumentGeneratorTest.java

```diff
@@ -81,4 +87,28 @@ public void shouldPublishDocumentReadyToPrintEventHavingLetterDocumentBeforeClai
         verify(publisher)
             .publishEvent(new DocumentReadyToPrintEvent(claim, defendantLetterDocument, sealedClaimDocument));
     }
+
+    @Test
+    public void shouldTriggerDocumentUploadEventForNonRepresentedClaim() {
```

Comment:
Prefer to call it `....CitizenClaim` instead of `...NonRepresentedClaim`

Reply:
NonRepresentedClaim makes sense to me and besides it is being used in several places already in the code.